sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/LifetimeActionRequest.java

```diff
@@ -25,10 +25,10 @@
 
     LifetimeActionRequest(LifetimeAction lifetimeAction) {
         action = new Action()
-            .actionType(lifetimeAction.actionType());
+            .setActionType(lifetimeAction.getActionType());
         trigger = new Trigger()
-            .daysBeforeExpiry(lifetimeAction.daysBeforeExpiry())
-            .lifetimePercentage(lifetimeAction.lifetimePercentage());
+            .daysBeforeExpiry(lifetimeAction.getDaysBeforeExpiry())
```

Comment:
should these methods have get and set prefixes?
`setDaysBeforeExpiry` and set `lifetimePercentage` ?